### ISSUE TITLE
Rename route’s data property in Webapp

### ISSUE THREAD

**Issue by latin-panda** [medic/cht-core#7043]
**Description:**
In order to add a css class wrapping a page (messages, reports, etc) and apply the correct styling, the route’s data object needs a `tab` property. However, not all are tabs, like Testing or About pages.

<img width="381" alt="Screen Shot 2021-03-30 at 11 39 01 am" src="https://user-images.githubusercontent.com/66472237/112934698-87fd5280-914c-11eb-963a-2b9ee551a1ae.png">

**Suggestion:**
Rename the property: it will avoid confusion and save time on trying to understand why a page that isn’t a tab has the property defined or why the page isn’t picking up the styles. 

seen @ Webapp, 3.11.

**Comment by dianabarsan** [comment]
The `reports.detail` isn't a `tab`, though. The base route (the parent) is the tab.

**Comment by dianabarsan** [comment]
Your desired improvement would be to change `tab` to `baseRoute` or similar? It still has to be a different property name compared to the child route.

**Comment by latin-panda** [comment]
Yes, as suggestion: rename `tab` for another name that address other type of content that needs the styling. 
Or another suggestion can be to change how the style is applied, if that works better. 
Gotta analyse the code on development.

**Comment by santhosh-7777** [comment]
@dianabarsan is this issue still active ?

**Comment by andrablaj** [comment]
@santhosh-7777 hi, and thank you so much for your continued interest in contributing! I noticed you currently have [4 issues](https://github.com/orgs/medic/projects/367/views/4) assigned to you. To make sure everyone has a fair chance to contribute, we try to keep work-in-progress items to a minimum per contributor. Would you mind unassigning yourself from any issues you're not actively working on before picking up this one? Really appreciate it — thanks!
